runtime/criusupport/criusupport.cpp

```diff
@@ -376,16 +392,41 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			goto wakeJavaThreads;
 		}
 
+		Trc_CRIU_prepare_internal_hooks(currentThread);
+		vmFuncs->acquireExclusiveVMAccess(currentThread);
+		vm->memoryManagerFunctions->j9mm_iterate_all_objects(vm, PORTLIB, 0, objectIteratorCallback, vm->checkpointState.hookRecords);
```

Comment:
`j9mm_iterate_all_objects` and `runInternalJVMCriuHooks` can be combined into a single call in criu helpers

Reply:
Moved `j9mm_iterate_all_objects()` into `runInternalJVMCriuHooks(. FALSE)` case.